numpy/lib/polynomial.py

```diff
@@ -479,10 +479,10 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
         coefficients for `k`-th data set are in ``p[:,k]``.
 
     residuals, rank, singular_values, rcond
-        Present only if `full` = True.  Residuals of the least-squares fit,
-        the effective rank of the scaled Vandermonde coefficient matrix,
-        its singular values, and the specified value of `rcond`. For more
-        details, see `linalg.lstsq`.
+        Present only if `full` = True.  Residuals is sum of squared residuals
+        of the least-squares fit, the effective rank of the scaled Vandermonde
+        coefficient matrix, its singular values, and the specified value of
+        `rcond`. For more details, see `linalg.lstsq`.
```

Comment:
@mattip - these attempted links to parameters should raise sphinx warnings, and ideally the docs build test should fail on those?

Reply:
In order for sphinx to catch these, we need to run with `-n`, which currently spits out over 1000 warnings. xref gh-14699 which fixes the common and non-controversial ones, and issue gh-14699 for tracking progress

Reply:
Oh, right, indeed. I assumed without much thinking that this was turned on already, sorry for the noise then. 
If an extra pair of hand would help, let me know, messing with sphinx is well within my comfort zone.

Reply:
a review of #14699 would be helpful, and continuation of the work would be great

Reply:
These shouldn't raise warnings, usage is correct here. At least, this is what the numpydoc spec says. And the rendering can be controlled, see https://github.com/numpy/numpydoc/issues/233